models/double_entry_transactions/int_quickbooks__sales_receipt_double_entry.sql

```diff
@@ -17,6 +17,12 @@ sales_receipt_lines as (
     from {{ ref('stg_quickbooks__sales_receipt_line') }}
 ),
 
+accounts as (
+
+    select *
+    from {{ ref('stg_quickbooks__account') }}
+),
+
```

Comment:
Is this necessary? It seems like this may be a holdover from the previous approach. If that is the case, can we remove this cte from the model.

Reply:
Removed.